docs/examples/google-cloud-function.ts

```diff
@@ -0,0 +1,24 @@
+import { Telegraf } from 'telegraf'
+
+const { BOT_TOKEN, FUNCTION_NAME, PROJECT_ID, REGION } = process.env
+
+if (BOT_TOKEN === undefined) {
+  throw new TypeError('BOT_TOKEN must be provided!')
+}
+
+const bot = new Telegraf(BOT_TOKEN)
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+bot.telegram.setWebhook(
+  `https://${REGION!}-${PROJECT_ID!}.cloudfunctions.net/${FUNCTION_NAME!}`
+)
+
+bot.command('hello', (ctx) => ctx.reply('Hello, friend!'))
+
+export const botFunction = async (req: any, res: any) => {
```

Comment:
I'm not sure with the platform. I assume this exported function is then used by the google cloud thingy?

Two questions here:

- Can we provide typings for the arguments? If this example is copied over users might find these typings helpful. But they are probably not as simple and would require an additional import?

- I would expect functions to be declared like functions and not like variables? Or is there any reason for this I am missing here? Or just personal preference?
```suggestion
export async function botFunction(req: any, res: any) {
```

Reply:
Here function syntax is purely a preference, not a problem, so I left it as-is.

More accurate parameter types would require a `devDependency` on `@types/express`.

Reply:
Maybe just add a comment for this requirement then? Seems better than `any` to me. (I really don't like seeing `any`, especially in code meant to be copied.)
```ts
// You have to `npm install --save-dev @types/express` to use this example
import {Request, Response} from 'express'
```

But I assume the CI will then be annoying about unknown types…?